src/style.css

```diff
@@ -1,3 +1,60 @@
 body {
   margin: 0;
+  padding: 0;
+  --main-color: #00acdc;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 2px 3px 3px rgba(141, 141, 141, 0.73);
+}
+
+.header__logo {
+  display: block;
+}
+
+.nav__items {
+  display: flex;
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__link {
+  position: relative;
+  display: flex;
+  align-items: center;
+  height: 60px;
+  text-decoration: none;
+  color: #000;
+  text-transform: uppercase;
+  font-weight: 500;
+}
+
+.nav__item:not(:first-child) {
+  margin-left: 20px;
+}
+
+.nav__link:hover {
```

Comment:
You should merge styles because as you can see  `nav__link:hover` and `is-active` have the same color